examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -246,4 +257,21 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
       ),
     );
   }
+
+  Widget buildIosButton() {
```

Comment:
The capitalisation should be "iOS". If that doesn't fit, make it private and use "Cupertino".